dali/pipeline/operators/arithmetic/expression_tree.cc

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I think these functions require/deserve a decent set of tests to check how they handle different kinds of string formatting. Or we want to assume that only a valid one will be provided (generated automatically - not handcrafted) and we don't have to be fault-proof.

Reply:
As of now, the assumption is, this is strictly internal representation.
The error messages may be helpful during development.